lms/templates/instructor/instructor_dashboard_2/certificates.html

```diff
@@ -92,6 +92,24 @@
         %endif
     % endif
 
+    <hr>
+    <div class="start-certificate-regeneration">
+        <h2>${_("Regenerate Certificates")}</h2>
```

Comment:
Add information paragraph after header: 
"Select one or more students below using your mouse and ctrl or command key."